dbux-data/src/dataProviderUtil.js

```diff
@@ -25,6 +26,18 @@ export default {
     return dp.collections.staticProgramContexts.getById(programId)?.filePath || null;
   },
 
+  /** @param {DataProvider} dp */
+  getProgramModuleName(dp, programId) {
+    const programContext = dp.collections.staticProgramContexts.getById(programId);
+
+    if ('_moduleName' in programContext) {
```

Comment:
No need to `return` twice

```js
if (!x) {
  x = ...;
}
return x;
```

Reply:
@MiccWan  (I don't know, if you see these messages, if the PR is approved anyway :D )